float32             grid_spacing
bool		    clear_buffer
--- 
int32               pattern_count
bool                success
geometry_msgs/Pose  pose
